frontend/src/styles/Landing.scss

```diff
@@ -0,0 +1,51 @@
+@import "./constants";
+
+.hero-container {
+  position: relative;
+  height: 70vh;
+  width: 100vw;
+  // margin: 0em 0em 7em 0em;
+  margin: 0em;
+  display: flex;
+  justify-content: center;
+  flex-direction: column;
+  text-align: center;
+  @include overlay;
+
+  .hero-img {
```

Comment:
I usually don't like to nest selectors. But that's debatable. I actually run into issues if I need to deal with nested selectors.
Imagine that we move the element that uses this `hero-img` class to outside the `hero-container`? It will completely lose the style.
Also there's the issue of selector specificity that can cause issues.
Generally I'd recommend dealing with no nesting, it can make your life easier.

Reply:
Yes this is a valid point, I took them out now 🙌